README.md

```diff
@@ -1,9 +1,12 @@
 # apollo-fetch [![npm version](https://badge.fury.io/js/apollo-fetch.svg)](https://badge.fury.io/js/apollo-fetch) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://www.apollostack.com/#slack)
 
 
-`apollo-fetch` is a lightweight client for GraphQL requests that supports middleware and afterware that modify requests and responses.
+`apollo-fetch` is a lightweight client for single and batched GraphQL requests that supports middleware and afterware that modify requests and responses.
```

Comment:
I think this dilutes the description. Batching is a feature, no need to mention it in the tagline.